k9mail/src/main/java/com/fsck/k9/preferences/DeletePolicy.java

```diff
@@ -0,0 +1,34 @@
+package com.fsck.k9.preferences;
+
+import android.util.Log;
+
+import com.fsck.k9.Account;
+import com.fsck.k9.K9;
+import com.fsck.k9.mail.ServerSettings;
+import com.fsck.k9.mail.store.imap.ImapStore;
+import com.fsck.k9.mail.store.pop3.Pop3Store;
+import com.fsck.k9.mail.store.webdav.WebDavStore;
+
+/**
+ * Defaults for mail deletion.
+ */
+public class DeletePolicy {
```

Comment:
OK, I will do the enums as a separate pull request, once that's in master I'll merge it in here.

On this branch, I'll create a new anemic AccountCreationService and the defaultPolicy logic can go in there, pending the rest of the code being moved in too.

RE naming: there is a com.fsck.k9.service package (MailService etc.). These are Android Service classes, so I don't want AccountCreationService to get confused for one of them. Can you think of another name?